internal/commands/log.go

```diff
@@ -0,0 +1,169 @@
+package commands
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli/v3"
+	"go.uber.org/zap"
+
+	"github.com/hathora/ci/internal/commands/altsrc"
+	"github.com/hathora/ci/internal/output"
+	"github.com/hathora/ci/internal/sdk"
+	"github.com/hathora/ci/internal/setup"
+	"github.com/hathora/ci/internal/workaround"
+)
+
+var (
+	minTailLines = 1
+	maxTailLines = 5000
+)
+
+var Log = &cli.Command{
+	Name:  "log",
+	Usage: "view live process logs",
+	Flags: subcommandFlags(followFlag, processIDFlag, tailLinesFlag),
+	Action: func(ctx context.Context, cmd *cli.Command) error {
+		log, err := ProcessLogConfigFrom(cmd)
+		if err != nil {
+			//nolint:errcheck
+			cli.ShowSubcommandHelp(cmd)
+			return err
+		}
+
+		if err := log.Validate(); err != nil {
+			//nolint:errcheck
+			cli.ShowSubcommandHelp(cmd)
+			return err
+		}
+
+		log.Log.Debug("Getting build server logs...")
+
+		res, err := log.SDK.LogV1.GetLogsForProcess(
+			ctx,
+			log.ProcessID,
+			log.AppID,
+			sdk.Bool(log.Follow),
+			sdk.Int(log.TailLines))
+
+		if err != nil {
+			return fmt.Errorf("failed to get logs: %w", err)
+		}
+
+		err = output.StreamOutput(res.Stream, os.Stderr)
```

Comment:
I think in this instance it'd good to stream the output to `os.Stdout` so that it's easy to pipe to `grep`

Reply:
Tested that piping to grep worked